cli/cmd_install.go

```diff
@@ -23,7 +26,11 @@ func runInstall() {
 	var exitCode int
 	defer func() { os.Exit(exitCode) }()
 
-	b, err := bob.Bob()
+	cache, err := nix2.NewFileCacheStore()
+	errz.Fatal(err)
+	nix := bob.NewNix(cache)
```

Comment:
This should be the default setting when bob is initialized, so it's only required to pass a different nix object during testing.

Please assure the default cache is set here https://github.com/benchkram/bob/pull/70/files#diff-628c0fa24ba19b12a7820b6d10351169e0cfaaa6473f35582a5b71c7ffa98a3dL54